app/controllers/categories_controller.rb

```diff
@@ -196,8 +196,8 @@ def update
       category_params.delete(:custom_fields)
 
       # properly null the value so the database constraint doesn't catch us
-      category_params[:email_in] = nil if category_params[:email_in]&.blank?
-      category_params[:minimum_required_tags] = 0 if category_params[:minimum_required_tags]&.blank?
+      category_params[:email_in] = nil if category_params[:email_in].blank?
+      category_params[:minimum_required_tags] = 0 if category_params[:minimum_required_tags].blank?
```

Comment:
Adjacent code cleanup. No need to use lonely operator when calling `#blank?` since it's defined one `nil` as well.

![](https://media0.giphy.com/media/26Ff2bTANl3s1UDOU/giphy.gif?cid=ecf05e470cw1o54op35iqxkbxgph3p4gfnr8znz9jnfihzns&ep=v1_gifs_search&rid=giphy.gif&ct=g)

Reply:
Yeah this is a weird oversight...also never heard this called the "lonely operator" :laughing: